<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWidget</class>
 <widget class="QWidget" name="CMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PurrFX reverse player</string>
  </property>
  <layout class="QVBoxLayout" name="layMain">
   <item>
    <layout class="QHBoxLayout" name="layFile">
     <item>
      <widget class="QLabel" name="lblFile">
       <property name="text">
        <string>Loaded NSF file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFile">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoad">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layOptions">
     <item>
      <widget class="QLabel" name="lblTrack">
       <property name="text">
        <string>Track:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbTrack">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTime">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbTime">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layControl">
     <item>
      <widget class="QPushButton" name="btnPlay">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
